src/ensureChildrenIsValid.js

```diff
@@ -0,0 +1,17 @@
+import isDOMElement from './isDOMElement';
+import isFunction from './isFunction';
+
+/**
+ * Raise an error if "children" isn't a single DOM Element or a function
+ *
+ * @param {React.element|null} children
```

Comment:
You should add `Function` (or whatever the jsdoc type is called) to the list here.